src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -92,7 +93,7 @@ @interface GRPCCall () <GRXWriteable>
 @implementation GRPCCall {
   dispatch_queue_t _callQueue;
```

Comment:
I rewrote the comment to match my changes. I think some improvements can still be made, such as starting the call with a single batch that does `SEND_METADATA`, `RECV_METADATA`, and `RECV_MESSAGE`, but I think it would be better to have a significant reworking of `GRPCCall` in another pull request.